whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -680,8 +679,13 @@ private static void createVCard(final Entity entity, final RpslObject rpslObject
                 .addOrg(rpslObject.getValuesForAttribute(ORG))
                 .addGeo(rpslObject.getValuesForAttribute(GEOLOC));
 
-        RDAP_VCARD_REDACTED_ATTRIBUTES.forEach(attributeType -> entity.getRedactedRpslAttrs()
-                .addAll(rpslObject.findAttributes(attributeType)));
+        if (!filtered){
+            builder.addEmail(rpslObject.getValuesForAttribute(E_MAIL));
+            entity.getRedactedRpslAttrs().addAll(rpslObject.findAttributes(NOTIFY));
+        } else {
+            List.of(NOTIFY, E_MAIL).forEach(attributeType -> entity.getRedactedRpslAttrs()
```

Comment:
Why is NOTIFY in the redacted list? It's used for Whois update notifications and not intended as a general contact address

Reply:
I remember that we were redacting NOTIFY because of personal information :S 
When we implemented the Redaction task.
I will remove NOTIFY then